me.leucoso.Grock.appdata.xml

```diff
@@ -0,0 +1,65 @@
+  <?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>me.leucoso.Grock.desktop</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0-or-later</project_license>
+  <name>Grock</name>
+  <summary>Displays geological maps of the UK.</summary>
+	<description>
+	  <p>Display geological maps of the UK, using data from the British Geological Survey.
+	    Allows rock unit names, types and ages to be found easily. Linear features and UK-wide
+	    gravitational and magnetic anomaly maps can also be shown.</p>
+	</description>
+  <categories>
+    <category>Geology</category>
+    <category>Geoscience</category>
+    <categroy>Maps</categroy>
+    <category>Geography</category>
+    <category>DataVisualization</category>
+    <category>Education</category>
+    <category>Science</category>
+    <category>GTK</category>
+  </categories>
```

Comment:
The categories from the appdata & desktop files are merged together at a later stage of the build. You can just mention them on the desktop file.